opentech/settings/production_heroku.py

```diff
@@ -0,0 +1,36 @@
+import os
+
+import django_heroku
+import sentry_sdk
+
+from sentry_sdk.integrations.django import DjangoIntegration
+
+from .base import *  # noqa
+
+# Disable debug mode
+DEBUG = False
+
+# Configuration from environment variables
+# Alternatively, you can set these in a local.py file on the server
```

Comment:
This is not true anymore, the local file is only loaded in `dev.py`.

What do you think of adding it to the new `production.py`?

https://github.com/OpenTechFund/opentech.fund/blob/7266d2d7e712172f7ddaa33a3972972a90622f85/opentech/settings/dev.py#L32-L35

Reply:
Great idea. Makes sense to me.